engine/components/IgeInputComponent.js

```diff
@@ -209,35 +209,76 @@ var IgeInputComponent = IgeEventingClass.extend({
 
 		// Define event functions and keep references for later removal
 		this._evRef = {
-			mousedown: function (event) { event.igeType = 'mouse'; self._rationalise(event, undefined, true); self._mouseDown(event); },
-			mouseup: function (event) { event.igeType = 'mouse'; self._rationalise(event); self._mouseUp(event); },
-			mousemove: function (event) { event.igeType = 'mouse'; self._rationalise(event); self._mouseMove(event); },
-			mousewheel: function (event) { event.igeType = 'mouse'; self._rationalise(event); self._mouseWheel(event); },
-
-			touchmove: function (event) { event.igeType = 'touch'; self._rationalise(event, true); self._mouseMove(event); },
-			touchstart: function (event) { event.igeType = 'touch'; self._rationalise(event, true); self._mouseDown(event); },
-			touchend: function (event) { event.igeType = 'touch'; self._rationalise(event, true); self._mouseUp(event); },
-
-			contextmenu: function (event) { event.preventDefault(); event.igeType = 'mouse'; self._rationalise(event); self._contextMenu(event); },
-
-			keydown: function (event) { event.igeType = 'key'; self._rationalise(event); self._keyDown(event); },
-			keyup: function (event) { event.igeType = 'key'; self._rationalise(event); self._keyUp(event); }
+			mousedown: function (event) {
+				event.igeType = 'mouse';
+				self._rationalise(event, undefined, true);
+				self._mouseDown(event);
+			},
+			mouseup: function (event) {
+				event.igeType = 'mouse';
+				self._rationalise(event);
+				self._mouseUp(event);
+			},
+			mousemove: function (event) {
+				event.igeType = 'mouse';
+				self._rationalise(event);
+				self._mouseMove(event);
+			},
```

Comment:
what you think, should we check these events if they are used or not for this PR? for example for 'mousemove' event looking like listeners for it are in unused js files
![image](https://user-images.githubusercontent.com/62058278/176230268-52e7539e-0e85-4c3a-bd25-c63a99bcea95.png)

Reply:
unused code is not an issue at the moment, but in case it might be used is some rare case we risk stuff breaking, so would not go that deep and keep it in